src/pages/CreatePool.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useState } from "react"
+import { useTranslation } from "react-i18next"
+
+export default function CreatePool(): React.ReactElement {
+  const { t } = useTranslation()
+  const theme = useTheme()
+  const [tokenLists, setTokenLists] = useState<string[]>([""])
+
+  const handleAddTokenList = () => {
+    setTokenLists((prev) => [...prev, ""])
+  }
+  return (
+    <Container sx={{ pb: 5 }}>
+      <Paper>
+        <Box p={3}>
+          <Box>
+            <Typography variant="h1">{t("createPool")}</Typography>
+            <Typography mt={2}>{t("createPoolDescription")}</Typography>
+          </Box>
+          <Box mt={4}>
+            <Typography variant="subtitle1">{t("addPoolName")}</Typography>
+            <Divider />
+            <Typography my={2}>{t("addPoolNameDescription")}</Typography>
+            <Box display="flex" flexDirection={["column", "row"]}>
+              <Box flex={1} mt={2}>
+                <TextField
+                  size="medium"
+                  label="Pool Name"
+                  fullWidth
+                  sx={{ mr: [0, 1.5], flex: 1 }}
+                />
+              </Box>
+              <Box flex={1} mt={2}>
+                <TextField
+                  size="medium"
+                  label="Pool Symbol"
+                  fullWidth
+                  sx={{ ml: [0, 1.5], flex: 1 }}
+                />
+              </Box>
+            </Box>
+          </Box>
+
+          <Box mt={4}>
+            <Typography variant="subtitle1">{t("setParameters")}</Typography>
+            <Divider />
+            <Stack direction="row" spacing={3} mt={2}>
+              <Box flex={1}>
+                <Typography mb={2}>{t("setFeeDescription")}</Typography>
+                <TextField label={`${t("fee")} (%)`} fullWidth />
+              </Box>
+              <Box flex={1}>
+                <Typography mb={2}>{t("amplificationParameter")}</Typography>
+                <Typography mb={2}>
+                  {t("suggestedValue")}
+                  <li>{t("suggestedValue-1")}</li>
+                  <li>{t("suggestedValue-2")}</li>
+                  <li>{t("suggestedValue-3")}</li>
+                </Typography>
+                <TextField label="A parameter" fullWidth />
+              </Box>
+            </Stack>
+          </Box>
+
+          <Stack direction="row" spacing={3} my={4}>
+            <Box flex={1}>
+              <Typography variant="subtitle1">{t("createPoolType")}</Typography>
+              <Divider />
+              <Typography my={2}>{t("createPoolTypeDescription1")}</Typography>
+              <Typography mb={2}>{t("createPoolTypeDescription2")}</Typography>
+              <Typography mb={2}>{t("createPoolTypeDescription3")}</Typography>
+              <ToggleButtonGroup
+                value="basePool"
+                color="secondary"
+                size="large"
+                fullWidth
+              >
+                <ToggleButton value="usdMetaPool" size="large">
```

Comment:
Keep naming consistent. Here you have `usdMetaPool` and below it's `btcMetapool`. I'd suggest `metapool` as we usually refer to it as one word.

Reply:
Sorry. This file is not necessary on this PR.
This file is for the following PR. https://github.com/saddle-finance/saddle-frontend/pull/840